### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by brusherru**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+2.  Press that
+3.  Click here and there
+
+### Expected behavior
+
+What should happen?
+
+### Actual behavior
+
+What’s happen now?
```
Comment (avive) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Change to `what's happening`

**Review by avive — CHANGES_REQUESTED** [review]
Small grammar change. I think it is good to have an improved issue template.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+labels: ''
+assignees: ''
+
+---
+
+### Environment
+
+* Smapp version: 0.xx.x | develop
+* Node version: v0.x.xx <!-- Open settings screen, info section -->
+* OS: Windows 10 | macOS High Sierra (10.13) | Ubuntu 18.04
```
Comment (IlyaVi) [inline .github/ISSUE_TEMPLATE/bug_report.md]: add "Ubuntu 20.04"

**Comment by brusherru** [comment]
Done!

**Review by IlyaVi — APPROVED** [review]

**Pull request merged**